files/default/supervisor-ckan-harvest-gather.conf

```diff
@@ -0,0 +1,47 @@
+[unix_http_server]
+file=/var/tmp/supervisor.sock
+
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+; ====================================================================
+; Supervisor configuration for CKAN Harvest background gather consumer
+; ====================================================================
+
+; 1. Copy this file to /etc/supervisor/conf.d
+; 2. Make sure the paths below match your setup
+
+
+[program:ckan-harvest-gather]
```

Comment:
Can we combine this with the "fetch" config file?

Reply:
they are all independant queues and should have at least one per service. supervisord sock can handle all it seems.